lib/graphql/dataloader/loader.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+module GraphQL
+  class Dataloader
+    class Loader
+      class StateError < GraphQL::Error; end
+
+      def self.load(context, key, value)
+        dl = context[:dataloader]
+        loader = dl.loaders[self][key]
```

Comment:
Yes, i think there's got to be some better API for this. But I'd really like to keep dataloading `context`-aware so that we can trace it as part of the GraphQL request.